plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1024,9 +1024,20 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
     private Optional<Long> getSnapshotId(org.apache.iceberg.Table table, Optional<Long> snapshotId)
     {
         // table.name() is an encoded version of SchemaTableName
```

Comment:
I do not understand what this comment says. Maybe drop.

Reply:
me neither but it was there before, it was added by @findepi and it looks like it means something to him